src/getchar.c

```diff
@@ -194,7 +197,9 @@ get_recorded(void)
     char_u *
 get_inserted(void)
 {
-    return get_buffcont(&redobuff, FALSE);
+    size_t  len;
+
+    return get_buffcont(&redobuff, FALSE, &len);
```

Comment:
```suggestion
    return get_buffcont(&redobuff, FALSE, NULL);
```
And then in `get_buffcont()` check whether a pointer or NULL has been given

Reply:
No worries.